examples/quickstart_aws/tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
+envlist = py{38, 37}
+skipsdist = true
+
+[testenv]
+download = true
+passenv = CI_* COVERALLS_REPO_TOKEN BUILDKITE*
+deps =
+  -e ../../python_modules/dagster[test]
+  -e ../../python_modules/dagit
+  -e ../../python_modules/libraries/dagster-aws/
+  pandas
+  matplotlib
+  textblob
+  tweepy
+  wordcloud
+  ; -e .
```

Comment:
this is pretty bad i think because it can't really test the setup.py of this package